RecoTracker/IterativeTracking/python/LowPtQuadStep_cff.py

```diff
@@ -195,15 +198,20 @@
 
 
 # Final selection
-from RecoTracker.FinalTrackSelectors.TrackLwtnnClassifier_cfi import *
-from RecoTracker.FinalTrackSelectors.trackSelectionLwtnn_cfi import *
-lowPtQuadStep = TrackLwtnnClassifier.clone(
-    src = 'lowPtQuadStepTracks',
-    qualityCuts = [0.2, 0.425, 0.75],
+from RecoTracker.FinalTrackSelectors.TrackMVAClassifierPrompt_cfi import *
+lowPtQuadStep = TrackMVAClassifierPrompt.clone(
+     mva = dict(GBRForestLabel = 'MVASelectorLowPtQuadStep_Phase1'),
+     src = 'lowPtQuadStepTracks',
+     qualityCuts = [-0.7,-0.35,-0.15]
 )
 fastSim.toModify(lowPtQuadStep,vertices = "firstStepPrimaryVerticesBeforeMixing")
```

Comment:
please move this line right after pp_on_AA to support more fastSim mixes

Reply:
Moved fastSim.toModify after pp_on_AA. Also changed L216 highBetaStar_2018.toReplaceWith -> .toModify as it had been before.